Enterprise-CMCS/eAPD
# Tforkner/4555 budget ffp read only

Resolves #4555 

### Description
Updated Budget and FFP read-only for MMIS. Added default messages if there are no state staff/other state expenses/private contractor costs. Added tests and cleaned up linting issues.

### Chromatic Link

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| an APD | there is no other funding | it doesn't have other funding in the list | jest |
| an APD | there is other funding | other funding appears in the list | jest |
| a HITECH APD | displaying Budget and FFP for Activity 1 | Key State Personnel are listed with State Staff | jest |
| a MMIS APD | displaying Budget and FFP | doesn't list Key State Personnel | jest |
| an APD | there is no State Staff/Other State Expenses/Private Contract| it display the default message | jest |

### Steps to manually verify this change

1. open a HITECH APD
2. navigate to Export and Submit
3. click on Continue to review
4. scroll down to the Budget and FFP section of Activity 1
5. verify that it displays like it did before

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when QA has


### This pull request can be merged when
